packages/trigger-client/src/TriggerClient.ts

```diff
@@ -83,6 +83,7 @@ export class TriggerClient {
     );
 
     const executeParams: TriggerServerExecuteRequestByType['place_order'] = {
+      id: params.id?.toNumber() ?? null,
```

Comment:
We need to be clear that consumers can't specify anything greater than max safe int - otherwise `toNumber()` will fail and just give an incorrect ID. Ideally BE can parse both strings and numbers so we can just convert to string here